MSAL/test/integration/native_auth/end_to_end/reset_password/MSALNativeAuthResetPasswordEndToEndTests.swift

```diff
@@ -47,7 +47,7 @@ final class MSALNativeAuthResetPasswordEndToEndTests: MSALNativeAuthEndToEndBase
             return
```

Comment:
Overall I see very little tests for the code written, should there be something? Maybe in the PublicClient tests?

Reply:
True. This PR is focused on the public interface update, for this reason there aren't many tests. 
New tests will be added in the next PR that contains the business logic.